OpenRA.Mods.Common/Activities/CaptureActor.cs

```diff
@@ -125,7 +125,7 @@ void DoCapture(Actor self, Captures captures)
 					t.OnCapture(enterActor, self, oldOwner, self.Owner, captures.Info.CaptureTypes);
 
 				if (self.Owner.RelationshipWith(oldOwner).HasRelationship(captures.Info.PlayerExperienceRelationships))
-					self.Owner.PlayerActor.TraitOrDefault<PlayerExperience>()?.GiveExperience(captures.Info.PlayerExperience);
+					self.Owner.PlayerActor.TraitOrDefault<PlayerExperience>()?.GiveExperience(captures.Info.PlayerExperience + captures.Info.PlayerExperiencePercentage * (enterActor.Info.TraitInfo<ValuedInfo>()?.Cost ?? 0) / 100);
```

Comment:
This should probably use `TraitInfoOrDefault`?

Reply:
Yes, fixed